buildh/cli.py

```diff
@@ -117,21 +120,30 @@ def parse_cli():
     Handle the user parameters from the command line.
     """
 
+    # Retrieve list of supported lipids
+    lipids_files = [ f for f in PATH_JSON.iterdir() if f.is_file()]
+    lipids_topH = lipids.read_lipids_topH(lipids_files)
+    lipids_supported_str = ", ".join(lipids_topH.keys())
+
     message = """This program builds hydrogens and calculate the order
     parameters
     (OP) from a united-atom trajectory. If -opx is requested, pdb and xtc
     output files with hydrogens are created but OP calculation will be slow.
     If no trajectory output is requested (no use of flag -opx), it uses a
     fast procedure to build hydrogens and calculate the OP.
     """
-    parser = argparse.ArgumentParser(description=message)
+    epilog = f"The list of supported lipids (-l option) are: {lipids_supported_str}."
+    parser = argparse.ArgumentParser(description=message,
+                                     epilog=epilog)
     # Avoid tpr for topology cause there's no .coord there!
     parser.add_argument("topfile", type=isfile,
                         help="Topology file (pdb or gro).")
     parser.add_argument("-x", "--xtc", type=isfile,
                         help="Input trajectory file in xtc format.")
     parser.add_argument("-l", "--lipid", type=str, required=True,
                         help="Residue name of lipid to calculate the OP on (e.g. POPC).")
+    parser.add_argument("-t", "--lipid_topology", type=isfile, nargs='+',
```

Comment:
I guess you name it `-t`  for topology? Is it possible to have 2 letters for the short version, e.g. `-tl` ?

Reply:
yes indeed.